libtransmission/blocklist.cc

```diff
@@ -49,10 +50,67 @@ void BlocklistFile::ensureLoaded() const
         return;
     }
 
-    auto range = IPv4Range{};
-    while (in.read(reinterpret_cast<char*>(&range), sizeof(range)))
+    auto file_size = std::filesystem::file_size(filename_);
```

Comment:
`std::filesystem` is unavailable many platforms. Please use `tr_sys_path_get_info()` instead

Reply:
Done :) !